Automattic/jetpack
# Add/growth upsells to social and stats interstials

## Proposed changes:

* Add growth class
* Add growth upsell to Stats and Social interstitial

### Other information:


## Jetpack product discussion

P2: pbNhbs-bxX-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Go to `/wp-admin/admin.php?page=my-jetpack#/add-social` and ensure you see the Growth upsell
![image](https://github.com/user-attachments/assets/02a464aa-be33-42c1-94aa-0d6f189bc1b5)
3. Click on the CTA and make sure it takes you to checkout with Growth yearly in the cart
4. Do the same with `/wp-admin/admin.php?page=my-jetpack#/add-stats`
![image](https://github.com/user-attachments/assets/67eb015a-14f7-468d-a876-e39c52978d54)
